main.go

```diff
@@ -74,6 +81,7 @@ func main() {
 				ExcludeGlobs:    c.StringSlice("glob-exclude"),
 				RunDir:          c.String("run"),
 				CachedBuild:     c.String("cache"),
+				Args:            flag.ShlexStrings(c.Generic("args")),
```

Comment:
I don't find this very attractive, but was not able to come up with a shorter solution to hide the ugly conversion from interface{} to []string.

Reply:
Looks good to me.